packages/styles/src/navigation.scss

```diff
@@ -1023,4 +1030,56 @@ $navListContainerSubmenu: #{$block}__list-container--submenu;
       --fdNavigation_Link_Back_Indicator_Icon: "\e1ed";
     }
   }
+
+  @include _fd-vertical-nav() {
+    .#{$block} {
+      &__item {
+        &--with-expander {
```

Comment:
why is this nesting needed? Isn't this just  #{$block}__item--with-expander or $navItemWithExpander?